packages/cdk/scroll/src/virtual/VirtualScroll.tsx

```diff
@@ -24,32 +24,36 @@ export default defineComponent({
   name: 'CdkVirtualScroll',
   props: virtualListProps,
   setup(props, { expose, slots }) {
-    const useVirtual = computed(() => props.virtual && props.height > 0 && props.itemHeight > 0)
+    const useVirtual = computed(() => props.virtual && props.itemHeight > 0)
```

Comment:
```suggestion
    const mergedVirtual = computed(() => props.virtual && props.itemHeight > 0 &&  (props.height > 0 || props.autoHeight))
```
另外我想把 virtual 改成 disabled, 就是废弃掉 virtual, 用 disabled 代替